### DISCUSSION TITLE
Accessibility

### DISCUSSION THREAD

**Discussion by wareid** [lukevella/rallly#477] [Ideas]
Hi there! 

I came across Rallly while looking for an alternative to Doodle. Doodle was something we have used a lot in the standards community to schedule meetings for groups across many time zones. We recently had to abandon it due to Doodle making changes that broke the limited accessibility the platform had, and their decision to not prioritize accessibility as a product goal. 

I have played around with Rallly and am so far impressed with it's UX and very clean design, however I notice that there is nothing in regards to accessibility, particularly for screen reader users. I wonder what your plans are in this respect, because in terms of implementation, I actually think you are not far from being accessible, it just requires some additional work. 

I understand this is an open source project, so am happy to try and help where I can, but I hope this is a priority or area of interest for this tool.

**Comment by lukevella** [comment]
Hey @wareid, thanks for opening this discussion. Accessibility is a growing concern for the project having been a bit ignored in the past for the sake of moving forward. However I have gained a much better understanding of its importance now and as Rallly's user base grows it is becoming a higher priority.

So the plan is to update small parts of the app at a time. The priority in which the parts get worked on is still determined by what features are being added but for each part that gets replaced it should be replaced by a more accessible version. The progress will be gradual and may seem slow but hopefully in time we will have something where accessibility can be advertised as a feature as well.

**Reply by lukevella** [reply]
Also, I want to add that I would really welcome opening issues or PRs if there are any clear and direct ways to improve things.

Something like, “Add alt tags to all images” is likely to get addressed and fixed quicker than “The poll component isn’t accessible”. I’m not an accessibility expert so being specific goes a long way :)

**Reply by wareid** [reply]
Luke, this is awesome to hear! 

I'll open up some actionable issues. I don't have the bandwidth to do a complete audit, but let me open up some issues for things I spotted and that are easy to address!
